tiledb/rest/capnp/query.cc

```diff
@@ -52,22 +53,14 @@ tiledb::sm::Status query_serialize(
   try {
     ::capnp::MallocMessageBuilder message;
     Query::Builder query_builder = message.initRoot<Query>();
-    tiledb::sm::Status status = query->capnp(&query_builder);
-
-    if (!status.ok())
-      return tiledb::sm::Status::Error(
-          "Could not serialize query: " + status.to_string());
+    RETURN_NOT_OK(query->capnp(&query_builder));
 
     serialized_buffer->reset_size();
     serialized_buffer->reset_offset();
 
-    uint64_t totalNumOfBytesInBuffers{
-        query_builder.getTotalNumOfBytesInBuffers()};
-    uint64_t totalNumOfBytesInOffsets{
-        query_builder.getTotalNumOfBytesInOffsets()};
-    uint64_t totalNumOfBytesInVarBuffers{
-        query_builder.getTotalNumOfBytesInVarBuffers()};
-
+    uint64_t total_fixed_len_bytes =
```

Comment:
Thinking on this do we need to track the length of buffers separately or could this just be total_buffer_length_bytes?

Reply:
I don't think we need these two values to be a part of the serialization process at all -- they are redundant.